### PR TITLE
3.12.3

### PR DESCRIPTION
**Pull request by 17blue17**

### CODE REVIEW

File: config/common/megastructures.cwt [hunk 1]
```diff
@@ -199,4 +199,30 @@ megastructure = {
...
+	dismantle_time = int
+	
+	## cardinality = 0..1
+	dismantle_potential = {
+		alias_name[trigger] = alias_match_left[trigger]
+	}
+	
+	## cardinality = 0..1
+	dismantle_possible = {
+		can_dismantle_megastructure = {
```
Comment (CaligulaCaesar) [inline config/common/megastructures.cwt:219]: Should be a trigger

File: config/common/species_consolidated.cwt [hunk 2]
```diff
@@ -208,29 +208,17 @@ species_class = {
 		## cardinality = 0..inf
 		<portrait_group>
 	}
+	# had to replace with normal triggers as government triggers would not work.  Had to make a special trigger at bottom for authority.
```
Comment (CaligulaCaesar) [inline config/common/species_consolidated.cwt:211]: I will take a look. It is a government trigger and we don't want CW Tools to think that the government trigger format is valid elsewhere.

File: config/enums.cwt [hunk 3]
```diff
@@ -390,4 +390,24 @@ enums = {
 		character
 		all
 	}
+	enum[variable_enum] = {
```
Comment (CaligulaCaesar) [inline config/enums.cwt:393]: This is surely just a vanilla bug where they check for some variables that don't exist.

**Review by CaligulaCaesar — APPROVED** [review]

File: config/enums.cwt (hunk 3 above)
Comment (17blue17) [reply config/enums.cwt:393]: The problem is that the local_size_option one is in a inline script which causes multiple errors in the common20_leader_subclasses.txt file which will now be permanently in error,   And the other one causes an astrial planes event file to go in to error.   I doubt Paradox will fix either of these in the future.   Things worked before you removed this enum from the trigger check variable.   Its too bad as I had things tested and working.

Comment (17blue17) [reply config/enums.cwt:393]: 17blue17 

The problem is that the local_size_option one is in a inline script which causes multiple errors in the common20_leader_subclasses.txt file which will now be permanently in error, And the other one causes an astrial planes event file to go in to error. I doubt Paradox will fix either of these in the future. Things worked before you removed this enum from the trigger check variable. Its too bad as I had things tested and working.

File: config/common/megastructures.cwt (hunk 1 above)
Comment (17blue17) [reply config/common/megastructures.cwt:219]: I understand it is a scripted trigger.   OK will fix with a new PR.

File: config/common/species_consolidated.cwt (hunk 2 above)
Comment (17blue17) [reply config/common/species_consolidated.cwt:211]: I did add a description on the two new triggers warning to only use in common species classes file.  It was on a separate change below this one.  So people should already be warned.

**Review by 17blue17 — COMMENTED** [review]
Replied to you questions below

**Pull request merged**